lib/bokken/accounts/social.ex

```diff
@@ -11,7 +11,7 @@ defmodule Bokken.Accounts.Social do
 
   @derive Jason.Encoder
   @primary_key {:name, Ecto.Enum,
-                values: [:scratch, :codewars, :github, :gitlab, :trello, :discord, :slack],
+                values: [:scratch, :codewars, :github, :gitlab, :trello, :discord, :slack, :codemonkey, :lightbot],
```

Comment:
Don't forget to use `is_socials_empty?` on the ninja changeset, just like I did with the mentor's one.